code-challenges.js

```diff
@@ -54,8 +163,29 @@ var refactorTester1 = true
 var refactorTester2 = 42
 var refactorTester3 = "IAmACodingMaster"
 
-
-
+//create a function named noVowelOnly
+//takes in a string
+const noVowelOnly = (str) => {
+    //if it is a string it will return a string with no vowels
+    if (typeof str === "string"){
+        //create a new variable that takes the string an makes it an array
+        strArray = str.split("")
+    //use filter method to iterate though the newly formed array and return it
+     return strArray.filter(value =>{
```

Comment:
try to be consistent with indentation. This logic is nested in the if statement and so it should be indented further (not on the same level as the `if`